Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -144,7 +150,7 @@ class SendViewController: UIViewController {
     private func fetchExchangeRate() {
         self.retryFetchExchangeRateButton.isHidden = true
         if self.exchangeRate == nil {
-            self.usdAmountLabel.text = "- USD"
+            self.usdAmountLabel.text = "0 USD"
```

Comment:
Please revert this, it would give the impression that the USD amount equivalent to the dcr amount in the amount text field is "0 USD" rather than unknown